screen/OrderData.xml.ftl

```diff
@@ -0,0 +1,172 @@
+              </OrderNumber>
+              <OrderDate>${order.placedDate!}</OrderDate>
+              <OrderStatus>
+                 <![CDATA[${part.partStatusId!}]]>
+              </OrderStatus>
+              <LastModified>${ec.l10n.format(orderHeaderAndPart.lastUpdatedStamp, 'yyyy-MM-dd HH:MM')}</LastModified>
+              <#assign shipmentMethod = ec.entity.find("moqui.basic.Enumeration").condition("enumId",part.shipmentMethodEnumId!).useCache(true).one()>
+              <ShippingMethod>
+                 <![CDATA[${shipmentMethod.description!}]]>
+              </ShippingMethod>
+              <#assign payment = ec.entity.find("mantle.account.payment.Payment").condition("orderId",order.orderId).condition("orderPartSeqId",part.id).list()>
+              <#if !payment.isEmpty()>
+                  <#assign paymentMethod = ec.entity.find("mantle.account.method.PaymentMethod").condition("paymentMethodId",payment[0].paymentMethodId!).one()>
+                  <#assign paymentDescription = ec.entity.find("moqui.basic.Enumeration").condition("enumId",paymentMethod.paymentMethodTypeEnumId!).useCache(true).one()>
+                  <PaymentMethod>
+                     <![CDATA[${paymentDescription.description!}]]>
+                  </PaymentMethod>
+              </#if>
+              <CurrencyCode>${order.currencyUom!}</CurrencyCode>
+              <OrderTotal>${orderHeaderAndPart.partTotal!}</OrderTotal>
+              <#--
+              <TaxAmount>XX</TaxAmount>
+              -->
+              <ShippingAmount>${part.shippingCost!}</ShippingAmount>
+              <CustomerNotes>
+                 <![CDATA[${part.shippingInstructions!}]]>
+              </CustomerNotes>
+              <InternalNotes>
+                 <![CDATA[${part.shippingInstructions!}]]>
+              </InternalNotes>
+              <Gift>${orderHeaderAndPart.isGift!}</Gift>
+              <GiftMessage>${orderHeaderAndPart.giftMessage!}</GiftMessage>
+              <Customer>
+                 <CustomerCode>
+                    <![CDATA[${order.customer_details.email!}]]>
+                 </CustomerCode>
+                 <BillTo>
+                    <Name>
+                       <![CDATA[${order.billing_details.address.toName!}]]>
+                    </Name>
+                    <#--
+                    <Company>
+                       <![CDATA[XX]]>
+                    </Company>
+                    -->
+                    <Phone>
+                       <![CDATA[${order.billing_details.phone.contactNumber!}]]>
+                    </Phone>
+                    <Email>
+                       <![CDATA[${order.customer_details.email!}]]>
+                    </Email>
+                 </BillTo>
+                 <ShipTo>
+                    <#if !shipAddress.isEmpty()>
+                        <#assign geoState = ec.entity.find("moqui.basic.Geo").condition("geoId",shipAddress.stateProvinceGeoId!).useCache(true).one()>
```

Comment:
Is it necessary to use useCache here? What is its importance?

Reply:
Yes for such entity finds where values can be repeated.

Reply:
That's great for performance then, thanks Siddharth!!